server/cmd/mmctl/commands/user.go

```diff
@@ -366,6 +366,8 @@ func init() {
 	ListUsersCmd.Flags().Bool("all", false, "Fetch all users. --page flag will be ignore if provided")
 	ListUsersCmd.Flags().String("team", "", "If supplied, only users belonging to this team will be listed")
 	ListUsersCmd.Flags().Bool("inactive", false, "If supplied, only users which are inactive will be fetch")
+	ListUsersCmd.Flags().Bool("roles", false, "If supplied, display user roles")
```

Comment:
I'm not convinced that we should go down that rabbit hole. Let's keep the rendering uniform. There is still the json option for people who want to parse it.